etools-searchable-multiselection-menu.html

```diff
@@ -494,6 +505,10 @@
             notify: true,
             value: []
           },
+          twoLines: {
```

Comment:
The attribute is not a label its just a suggestive attribute, so I think adding label to the name is a bit misleading since there is no assignment, it is just a boolean saying the options will be two lines.